### DISCUSSION TITLE
Variations

### DISCUSSION THREAD

**Discussion by th-art001** [lllyasviel/Fooocus#1187] [Ideas]
Hi! Thank you for Fooocus! 
I haven't been using it for a while, but I've noticed that "Subtle" variations are looks too weak, and "strong" are too... strong, i think. Can I affect this option? If not, so it would be great too have third button with an average value.
![Screenshot_221](https://github.com/lllyasviel/Fooocus/assets/59409653/20622f5d-8a60-47b1-a4a4-fecba8bc9a6f)
![varia](https://github.com/lllyasviel/Fooocus/assets/59409653/e752ec90-c727-47da-8431-cab1ff0c5adf)

**Comment by xhoxye** [comment]
You can set the level of noise reduction you want in developer mode.
subtle =0.5 , strong = 0.85
![QQ截图20231205092634](https://github.com/lllyasviel/Fooocus/assets/129571231/d036c75e-5925-4b13-bd3f-4b17d7ffabea)

**Reply by th-art001** [reply]
Hi, 0.6-0.65 this is exactly the value I need for "average"!
I'm sorry I didn't find it myself and thanks for the answer

**Comment by ke1ne** [comment]
> Hi, 0.6-0.65 this is exactly the value I need for "average"!
> I'm sorry I didn't find it myself and thanks for the answer

But adding of the AVERAGE button is still OK option

**Comment by PeterMcLennan** [comment]
Noob at Fooocus here. I want more control over img2img denoising strength, too.

How do I invoke "developer mode" ?

**Reply by xhoxye** [reply]
Advanced, Advanced, Developer Mode

**Comment by PeterMcLennan** [comment]
Thank you!  Now, I need to decipher those cryptic controls. :)

After eight months of InvokeIA and SD 1.5, Fooocus is a revelation for me.
Astonishing.
Thanks for making it happen for me.  It's making my winter here in Western
Canada far more fun!

cheeeers
Peter
